includes/wp-cli/class-events.php

```diff
@@ -634,6 +639,27 @@ private function delete_completed_events( $args, $assoc_args ) {
 
 		\WP_CLI::success( __( 'Entries removed', 'automattic-cron-control' ) );
 	}
+
+	/**
+	 * Delete events that have a particular schedule
+	 *
+	 * @param array $args Array of positional arguments.
+	 * @param array $assoc_args Array of flags.
+	 */
+	private function delete_events_by_schedule( $args, $assoc_args ) {
+		if ( 'null' === strtolower( $assoc_args['schedule'] ) ) {
+			$assoc_args['schedule'] = null;
+		}
+
+		$count = \Automattic\WP\Cron_Control\count_events_by_schedule( $assoc_args['schedule'] );
+
+		/* translators: 1: Event count */
+		\WP_CLI::confirm( sprintf( _n( 'Found %s single event to remove. Continue?', 'Found %s single events to remove. Continue?', $count, 'automattic-cron-control' ), number_format_i18n( $count ) ) );
```

Comment:
The use of `single event` is somewhat confusing here.

Reply:
9ab6261